pallets/did/src/did_details.rs

```diff
@@ -626,13 +651,28 @@ pub struct DidCreationDetails<T: Config> {
 	pub new_endpoint_url: Option<Url>,
 }
 
+// required because BoundedTreeSet does not implement Debug outside of std
+// FIXME: Can we derive Debug and use custom impl only in std?
+// #[cfg(not(feature = "std"))]
+impl<T: Config> fmt::Debug for DidCreationDetails<T> {
+	fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+		f.debug_tuple("DidCreationOperation")
```

Comment:
I don't know how long this will stay before finding a solution, but we should probably use `debug_struct` instead.